src/cli_driver.rs

```diff
@@ -1052,6 +1065,11 @@ fn main() {
              .help("How much chatter to print when running.")
              .possible_values(&["default", "minimal"])
              .default_value("default"))
+        .arg(Arg::with_name("outdir")
+             .long("outdir")
+             .short("o")
+             .value_name("OUTDIR")
+             .help("The output directory. Default: Same like INPUT."))
```

Comment:
Please format the default description as it appears in the other arguments; something like

`The directory in which to place output files. [default: the directory containing INPUT]`